stable/gocd/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ### 1.5.8
-[c8ac8202](https://github.com/kubernetes/charts/commit/c8ac8202):
+[5f028c73](https://github.com/kubernetes/charts/commit/c8ac8202):
```

Comment:
https://github.com/kubernetes/charts/commit/c8ac8202 - commit sha should change in the URL too?

Reply:
@sheroy Yes. :)

Might hold off on this till https://github.com/helm/charts/pull/9873 is merged, to reduce conflicts on it - and have @sheroy resolve it instead.

Reply:
This is quite a painful requirement ...